osgp/platform/osgp-adapter-domain-distributionautomation/src/main/java/org/opensmartgridplatform/adapter/domain/da/application/services/AdHocManagementService.java

```diff
@@ -99,4 +99,10 @@ public void handleGetDeviceModelResponse(final GetDeviceModelResponseDto getDevi
                 .build();
         this.webServiceResponseMessageSender.send(responseMessage, messageType);
     }
+
+    public void handleGetDataRequest(final String payload, final String deviceIdentification) {
+        LOGGER.info("Handle GET_DATA request for device: {} payload: {}", deviceIdentification, payload);
+        // TODO: further implement handling of GET_DATA messages
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1135)

Reply:
Intended as a pointer for later developement

Reply:
Intended as a pointer for further development using the MQTT protocol adapter